examples/cadence-samples/cadence-samples-pod.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
```

Comment:
Thanks for dockerizing this. We can do this in the samples repo and have CI to push image to dockerhub. This way, chart users can simply use it and avoid bunch of steps. I will track this separately.